### ISSUE TITLE
Bug 59398027: [React Native Gallery Preview]: On expanding/collapsing any button present in left navigation bar, Narrator remains silent.

### ISSUE THREAD

**Issue by vineethkuttan** [microsoft/react-native-gallery#684]
Build Details:
OS Version: 24H2 
OS Build: 26100.6584
App: React Native Gallery Preview
App Version: 1.0.18.0
AT Used: Narrator

Repro-Steps:
Launch the React Native Gallery-Preview app.
Now try to expand/collapse  any button present in left navigation bar
Now listen to the Narrator announcement.
Actual Result: 
When user expand/collapse any button present in left navigation bar, Narrator does not convey the changed state information. It remains silent. 

Expected Result: 
When expand/collapse any  button present in left navigation bar, Narrator should announce the <label name>, <control type>, <count& position>, <expand/collapse> state information

User Impact:
Users who rely on screen readers will not receive any feedback on the expand/collapse state of the left navigation bar if the Narrator remains silent

MAS Reference Link: [MAS 4.2.1 – Object Information](https://aka.ms/MAS4.2.1)

Trap ID: [1.4 - Uncomprehended Element](https://aka.ms/UncomprehendedElement)

“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”

**Comment by vineethkuttan** [comment]
@protikbiswas100 should merge his fix in 0.80
